README.md

```diff
@@ -28,6 +28,11 @@ Installer](https://github.com/gruntwork-io/gruntwork-installer). For example, to
 gruntwork-install --binary-name "kubergrunt" --repo "https://github.com/gruntwork-io/kubergrunt" --tag "v0.5.8"
 ```
 
+Install with Chocolatey:
```

Comment:
Since we don't officially maintain the choco package, the docs need a disclaimer:

```suggestion
### 3rd party package managers

Note that third-party Kubergrunt packages may not be updated with the latest version, but are often close. Please check your version against the latest available on the [Releases Page](/../../releases).

Chocolatey (Windows):
```